baselines/dasha/dasha/conf/base.yaml

```diff
@@ -0,0 +1,17 @@
+num_clients: 5
+num_rounds: 10000
+save_path: !!null
+local_address: !!null
+
+compressor:
+  _target_: dasha.compressors.RandKCompressor
+  number_of_coordinates: 1
+
+defaults:
+  - dataset: libsvm
+  - method: dasha
+  - model: linear_net_with_non_convex_loss
+
+hydra:
+  run:
+    dir: ${oc.env:HOME}/hydra_outputs/${now:%Y-%m-%d}/${now:%H-%M-%S}
```

Comment:
Out of curiosity, why is this introduced? This disables the (in my view) desirable Hydra behaviour of automatically creating an output directory for your experiment (where log and config are dumped). By default is created in the top-level dasha directory (i.e. sitting next to the .toml) and includes a sub-directory structure with date/time when the experiment is run. 

This is the recommended place to store the experiment outputs. (you can retrieve this directory's path with `save_path = HydraConfig.get().runtime.output_dir`)

Reply:
I'll return the expected behaviour.

> Out of curiosity, why is this introduced?

When I were developing and testing this project, I was using the repo in the read-only mode in a remote server. Locally, I could modify the repo, but remotely, where I executed the code, I could not do that. For instance, I could not produce outputs inside the repo. So I decided to save everything separate directories.